__tests__/app.test.js

```diff
@@ -140,3 +140,61 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("status:200 returns an articles array", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        body.articles.forEach((article) => {
+          expect(typeof article).toBe("object");
+        });
+      });
+  });
+  test("status:200 returned array should have 5 objects inside", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).toBe(5);
+      });
+  });
+  test("status:200 each object should have the correct properties and datatype", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        body.articles.forEach((article) => {
```

Comment:
^^ again - every test thaty ou use a .forEach in needs a length check -otherwise it'll autopass if it's being passed an empty array